tools/ios-tasks.js

```diff
@@ -307,23 +304,66 @@ async function generateReactPodspecAsync(
       podName => `ss.dependency         "${podName}"`
     ).join(`
     `);
-    let subspec = `
-  s.subspec "Expo" do |ss|
-    ss.dependency         "React/Core"
+    let subspec =
+ `s.subspec "Expo" do |ss|
+    ss.source_files     = "Expo/Core/**/*.{h,m}"
```

Comment:
```suggestion
    ss.source_files     = "Expo/Core/**/*.{h,m,mm}"
```
(if you think that future-proofing it will not cause trouble)